pallets/msa/src/lib.rs

```diff
@@ -837,14 +841,14 @@ impl<T: Config> Pallet<T> {
 	/// # Returns
 	/// * [`DispatchResult`]
 	/// # Errors
-	/// * [`ensure_valid_delegation`] Errors
+	/// * [`ensure_valid_schema_grant`] Errors
```

Comment:
@wilwade  this was breaking the doc build, I have fixed it here

Reply:
I thought I fixed all of the breaking parts. Oh well. Thanks!